### PR TITLE
Iterate over entrySet instead of keySet

### PR DESCRIPTION
**Pull request by gverm**
When both key and value are accessed.

Sonarqube links:
http://www.playonlinux.org:9000/issues#issues=AVx8gCMNaOc448AVSn53
http://www.playonlinux.org:9000/issues#issues=AVxD2WxM9U_f07q6H1GT
http://www.playonlinux.org:9000/issues#issues=AVw-tQRw9U_f07q6H1Cw
http://www.playonlinux.org:9000/issues#issues=AVw-tQLl9U_f07q6H1BU
http://www.playonlinux.org:9000/issues#issues=AVw-tQLm9U_f07q6H1BV
http://www.playonlinux.org:9000/issues#issues=AVw-tQLm9U_f07q6H1BW
http://www.playonlinux.org:9000/issues#issues=AVw-tQQc9U_f07q6H1Co

### CODE REVIEW

**Comment by plata** [comment]
ok to test

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/LibraryPanel.java [hunk 1]
```diff
@@ -109,13 +109,13 @@ public void setShortcutDTO(ShortcutDTO shortcutDTO) {
...
-            for (String shortcutKey : shortcutProperties.keySet()) {
-                final Label keyLabel = new Label(tr(unCamelize(shortcutKey)) + ":");
+            for (Map.Entry<String, Object> entry : shortcutProperties.entrySet()) {
+                final Label keyLabel = new Label(tr(unCamelize(entry.getKey())) + ":");
                 keyLabel.getStyleClass().add(CAPTION_TITLE_CSS_CLASS);
                 GridPane.setValignment(keyLabel, VPos.TOP);
                 this.gridPane.add(keyLabel, 0, i);
 
-                final Label valueLabel = new Label(shortcutProperties.get(shortcutKey).toString());
+                final Label valueLabel = new Label(entry.getValue().toString());
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/LibraryPanel.java]: @plata do we need a `tr(...)` here?

Comment (qparis) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/LibraryPanel.java]: I think so yeah

Comment (plata) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/LibraryPanel.java]: Yes, but should not be part of this PR.
